contracts/programs/gummyshop/src/state/mod.rs

```diff
@@ -0,0 +1,14 @@
+use anchor_lang::prelude::*;
+
+#[account]
+#[derive(Copy)]
+pub struct MarketplaceProperties {
+    // The address to which the marketplace will receive royalties
+    pub payment_receiver: Pubkey,
```

Comment:
Because we're account constrained, I think it could make sense to transfer the funds directly to this account and have the `payment_receiver` collect any lamports over rent exemption afterwards.

Reply:
Cool, will do this